README.md

```diff
@@ -1,12 +1,18 @@
-This is a Composer-based installer for the [Reservoir](https://github.com/acquia/reservoir) Drupal distribution. Welcome to the future!
+![Reservoir - Drupal distribution](https://raw.githubusercontent.com/acquia/reservoir-project/assets/reservoir.png)
 
-## Get Started
+# Reservoir: Composer installer
```

Comment:
Hm, "Composer" is the human name, so it's actually okay. Still, I'd expect it to use the "programmer" or "code" name.

Don't feel very strongly about this though.